sql/core/pom.xml

```diff
@@ -87,6 +87,16 @@
     </dependency>
 
     <dependency>
+      <groupId>org.apache.orc</groupId>
+      <artifactId>orc-core</artifactId>
+      <classifier>${orc.classifier}</classifier>
```

Comment:
Thank you, @rxin ! 
Currently, the `hive-storage-api` jar has something like `SearchArgument` and `PredicateLeaf`. Apache ORC is trying to become independent module like Parquet.